src/components/ChatEntry.js

```diff
@@ -1,22 +1,49 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
 
 const ChatEntry = (props) => {
+  const time = DateTime.fromISO(props.timeStamp);
+  const relative = time.toRelative();
+  const clickLikeButton = () => {
+    let updatedMessage = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+    };
+
+    props.onUpdate(updatedMessage);
+  };
+
+  let localRemote =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
```

Comment:
Tip: This is kind of brittle because if we swapped in a new JSON file (with new usernames) it would break.